lib/tasks/etl.rake

```diff
@@ -15,6 +15,19 @@ namespace :etl do
     end
   end
 
+  desc 'Delete existing metrics and Run Etl Master process across a range of dates'
+  task :rerun_master, %i[from to] => [:environment] do |_t, args|
+    from = args[:from].to_date
+    to = args[:to].to_date
+    (from..to).each do |date|
+      puts "Deleting existing metrics for #{date}"
```

Comment:
Can you think of a way to isolate each day in a transaction? This way if the process fails we don't leave inconsistencies